package.json

```diff
@@ -9,6 +9,7 @@
     "validate-schema": "node ./scripts/validate.js",
     "check-formatting": "prettier --check .",
     "format": "prettier --write .",
+    "hash": "node ./scripts/hash.js $1",
```

Comment:
This needs to not have the `$1`. `"hash": "node ./scripts/hash.js",` will do the trick.

NPM will pass through arguments as long as they're not `-` flags, and it does not do anything with the `$1` so far as I'm aware. I double checked in Windows and WSL Ubuntu as that's what I happened to be running when I saw this PR.